openupgrade_framework/odoo_patch/odoo/modules/loading.py

```diff
@@ -0,0 +1,110 @@
+import logging
+
+import odoo
+
+_logger = logging.getLogger(__name__)
+
+
+def load_modules(registry, force_demo=False, status=None, update_module=False):
+    with registry.cursor() as cr:
```

Comment:
Before you execute this operation, can you check if an update is going to take place? Otherwise there can be quite some overhead on large databases.

Reply:
Actually I was reading about slow count 2 days ago. [There is a much faster way.](https://www.citusdata.com/blog/2016/10/12/count-performance/) but still slow and also that article is nearly as old as Stefan's ref.

Reply:
Am I reading this wrong or is there no faster way that is actually exact? Count estimates are not really useful in this case.